### PR TITLE
symbiosis adds pufETH and zeta chain

### PR DESCRIPTION
**Pull request by allush**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/symbiosis-finance exports TVL: 
        
 
 
    ethereum                  5.19 M
    sei                       1.77 M
    arbitrum                  810.56 k
    polygon                   434.03 k
    era                       430.56 k
    tron                      373.27 k
    bsc                       269.96 k
    boba_bnb                  219.91 k
    merlin                    206.06 k
    linea                     182.99 k
    manta                     178.09 k
    zklink                    164.84 k
    ftn                       160.41 k
    scroll                    158.09 k
    rsk                       138.29 k
    mantle                    130.00 k
    base                      128.50 k
    optimism                  121.83 k
    boba                      115.40 k
    polygon_zkevm             112.13 k
    telos                     108.25 k
    metis                     106.62 k
    avax                      86.94 k
    blast                     77.82 k
    mode                      45.05 k
    arbitrum_nova             41.31 k
    core                      36.92 k
    taiko                     19.21 k
    kava                      10.33 k
    aurora                    4
    boba_avax                 0
    zeta                      0
    
    total                    11.82 M

File: projects/helper/coreAssets.json
Comment (g1nt0ki) [inline projects/helper/coreAssets.json:1]: can you revert the change to this file and add addresses directly to config.js? We try to limit the tokens on this file

Comment (allush) [reply projects/helper/coreAssets.json:1]: sure

**Comment by llamatester** [comment]
The adapter at projects/symbiosis-finance exports TVL: 
        
 
 
    ethereum                  4.97 M
    sei                       1.76 M
    arbitrum                  759.90 k
    era                       446.85 k
    polygon                   436.02 k
    tron                      404.82 k
    bsc                       363.40 k
    boba_bnb                  220.43 k
    merlin                    213.90 k
    linea                     186.48 k
    scroll                    166.80 k
    zklink                    165.61 k
    manta                     164.89 k
    ftn                       160.42 k
    rsk                       158.99 k
    mantle                    144.32 k
    base                      141.25 k
    optimism                  121.36 k
    boba                      115.47 k
    polygon_zkevm             110.25 k
    telos                     108.25 k
    metis                     105.97 k
    avax                      100.83 k
    mode                      52.62 k
    blast                     49.73 k
    arbitrum_nova             41.96 k
    core                      36.83 k
    taiko                     19.08 k
    kava                      10.34 k
    aurora                    4
    boba_avax                 0
    zeta                      0
    
    total                    11.74 M
